includes/class-newspack.php

```diff
@@ -254,7 +256,12 @@ public static function restrict_user_access( $current_screen ) {
 	 * @param WP_Screen $current_screen Current WP_Screen object.
 	 */
 	public function wizard_redirect( $current_screen ) {
-		$post_type_mapping = [];
+		$post_type_mapping = [
+			Emails::POST_TYPE => [
+				'base' => 'edit',
+				'url'  => esc_url( admin_url( 'admin.php?page=newspack' ) ),
```

Comment:
This is better than landing on the WP CPT post list for emails, but I wonder if we should add some more logic to redirect to the specific wizard page based on the email type? Engagement > Reader Activation for the verification email, and Reader Revenue > Emails for the Stripe reciept. If we're not consolidating the emails into a single wizard then it feels weird to be dumped back to the main Newspack dashboard page after closing out of the editor.

Reply:
@thomasguillot suggested elsewhere to use a hybrid approach to the email post type – link to specific emails where applicable, but also have a single "all emails" view. This will enable discoverability, but also allow to e.g. update branding on all emails from one place. 

I think the issue with the redirect is non-blocking, and the redirect can be updated to point to the "all emails" view once we have it.

Reply:
I'm okay with punting this to a new PR when we have an "All Emails" view.